flowmachine/tests/test_table.py

```
@@ -3,43 +3,60 @@
+    ],
+)
+def test_table_preflight(args):
+    with pytest.raises(PreFlightFailedException):
+        Table(**args).preflight()
 
-def public_schema_checked():
+
+def test_public_schema_checked():
     """Test that where no schema is provided, public schema is checked."""
-    t = Table("gambia_admin2")
+    t = Table("gambia_admin2", columns=["geom"]).preflight()
```

Ruff

Local variable `t` is assigned to but never used

Remove assignment to unused variable `t`

(F841)